### PR TITLE
new features: addQueue and universal worker

### PR DESCRIPTION
**Pull request by Santinell**

### CODE REVIEW

File: lib/worker.js [hunk 1]
```diff
@@ -22,9 +22,6 @@ function Worker(queues, options) {
     // This worker will only process jobs of this priority or higher
     this.minPriority = options.minPriority;
 
-    if (this.queues.length === 0) {
```
Comment (scttnlsn) [inline lib/worker.js]: Perhaps we check for `this.queues.length === 0 && !this.universal`?

Comment (Santinell) [inline lib/worker.js]: 1) var worker = client.worker(); //Empty worker. Can be filled dinamically. (this.queues.length === 0  this.universal === false)

2) var worker = client.worker('*'); //Universal worker. Can't add new queues there (this.queues.length === 1 &&  this.universal === true)

May be you mean second variant ?

Comment (scttnlsn) [inline lib/worker.js]: Good point

**Comment by scttnlsn** [comment]
This looks great.  Thanks!

**Pull request merged**